core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -454,23 +454,27 @@ class GroupCoordinator(val brokerId: Int,
     delayedOffsetStore.foreach(groupManager.store)
   }
 
+  def getPartitions(groupId: String): List[TopicPartition] = {
+      val group = groupManager.getGroup(groupId)
+      group.fold(List[TopicPartition]())(_.allOffsets.keySet.toList)
+  }
 
   def handleFetchOffsets(groupId: String,
-                         partitions: Seq[TopicPartition]): Map[TopicPartition, OffsetFetchResponse.PartitionData] = {
-    if (!isActive.get) {
-      partitions.map { topicPartition =>
-        (topicPartition, new OffsetFetchResponse.PartitionData(OffsetFetchResponse.INVALID_OFFSET, "", Errors.GROUP_COORDINATOR_NOT_AVAILABLE.code))}.toMap
-    } else if (!isCoordinatorForGroup(groupId)) {
+                         partitions: Seq[TopicPartition],
+                         apiVersion: Short): (Map[TopicPartition, OffsetFetchResponse.PartitionData], Errors) = {
```

Comment:
nit: slightly more natural if `Errors` is the first entry? Also, we don't need `apiVersion` anymore, right?